### PR TITLE
#3071 - update sbc-common-components, change event to listen to for s…

### PR DESCRIPTION
**Pull request by ochiu**
…ign in, fix tests

*Issue #:* /bcgov/entity#3071

*Description of changes:*
- Update SBC common components to 2.1.18 to address some errors during sign in
- Updated the event being listened to 'keycloak-session-ready' to address a looping issue
- Updated jest config to fix test errors when trying to transform updates to keycloak auth module

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/auth/Signin.vue [hunk 1]
```diff
@@ -62,6 +63,7 @@ export default class SigninView extends Vue {
         }
       }
     }
+    console.log('queryParam', paramConfig)
     return paramConfig
```
Comment (cameron-freshworks) [inline src/views/auth/Signin.vue]: This auto comment is an action i haven't seen before, very cool!

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "launchdarkly-js-client-sdk": "^2.17.0",
     "lodash": "^4.17.15",
     "regenerator-runtime": "^0.13.3",
-    "sbc-common-components": "2.1.8",
+    "sbc-common-components": "2.1.18",
```
Comment (cameron-freshworks) [inline package.json]: Things escalate quickly!

File: src/views/auth/Signin.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/views/auth/Signin.vue]: Nice that it catches this. If you don't the console.log then please remove it. If you do need it, add a lint override for it (eg, `// eslint-disable-next-line no-console` on previous line).

Comment (ochiu) [reply src/views/auth/Signin.vue]: Fixed

**Pull request merged**